nomad/alloc_endpoint_test.go

```diff
@@ -318,16 +319,26 @@ func TestAllocEndpoint_List_AllNamespaces_OSS(t *testing.T) {
 	defer cleanupS1()
 	codec := rpcClient(t, s1)
 	testutil.WaitForLeader(t, s1.RPC)
-
-	// Create the register request
-	alloc := mock.Alloc()
-	summary := mock.JobSummary(alloc.JobID)
 	state := s1.fsm.State()
 
-	err := state.UpsertJobSummary(999, summary)
-	require.NoError(t, err)
-	err = state.UpsertAllocs(structs.MsgTypeTestSetup, 1000, []*structs.Allocation{alloc})
-	require.NoError(t, err)
+	// two namespaces
```

Comment:
this previously didn't test multiple namespaces, so i expanded it while i was here